src/Drivers/StatamicTinkerwellDriver.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+use Tinkerwell\ContextMenu\Label;
+use Tinkerwell\ContextMenu\OpenURL;
+use Tinkerwell\ContextMenu\SetCode;
+use Tinkerwell\ContextMenu\Submenu;
+
+class StatamicTinkerwellDriver extends LaravelTinkerwellDriver
+{
+    public function canBootstrap($projectPath)
+    {
+        return file_exists($projectPath . '/vendor/statamic/cms/');
```

Comment:
I think you need to remove the trailing slash.
It's possible cms is a symlink, and having the slash would fail.